### DISCUSSION TITLE
Wrap Search Suggestions around top and bottom

### DISCUSSION THREAD

**Discussion by skube** [community/community#3559] [Discussions]
I ❤️ the **Search or jump to...** especially with the `/` shortcut. 
I 😫 having to press <kbd>↓</kbd> a bunch of times to get to the last item in the **Suggestions**

**Comment by PranamBhat** [comment]
Hi @skube  Can you please elaborate your issue ..

**Reply by skube** [reply]
Sure! 

When you use the location bar to search in a browser like Chrome, you'll get a dropdown of search suggestions. Using the keyboard, pressing the <kbd>↑</kbd>/<kbd>↓</kbd> keys will obviously move the highlighted selection. One very nice feature is that instead of pressing a series of <kbd>↓</kbd> arrows to move to the last item, you can instead press <kbd>↑</kbd> and immediately jump to the last item at the _bottom_ (as it wraps around).

The GitHub Search bar does not behave the same way and will stop at the first (or last item) in the search suggestion list.

**Comment by PranamBhat** [comment]
Yup! Rather than this, GitHub behaves like a search keyword feature.

For example, "dancers" will give youdancers emoji, and so on.. 

GitHub is damn cool! Happy to use this feature :)
